modules/solid_mechanics/include/materials/ADComputeIncrementalShellStrain.h

```diff
@@ -186,4 +199,7 @@ class ADComputeIncrementalShellStrain : public Material
   ADRealVectorValue _g2_b;
   ADRealVectorValue _g2_d;
   RankTwoTensor _unrotated_total_strain;
+  ADRealVectorValue _e1;
+  ADRealVectorValue _e2;
+  ADRealVectorValue _e3;
```

Comment:
I wanted to make this change, but ADComputeFiniteShellStrain.C also contains numerous _V1 and _V2 vectors with different indices of p and k, which is confusing. I think it would be better to leave this for later.

Reply:
OK, that makes sense. That's something that's probably worth considering in the future.